File: Practice/atcoder/ABC/014/src/c.py
# imos
n = int(input())

arr = [0 for _ in range(10 ** 6 + 2)]
for _ in range(n):
    a, b = map(int, input().split())
    arr[a] += 1
    arr[b + 1] -= 1
# 整列アンド探索フェーズ
ans = arr[0]   # n,a,b=1,0,0の場合があったので
for i in range(1, 10 ** 6 + 2):
    arr[i] += arr[i - 1]
    ans = max(ans, arr[i])
print(ans)

File: Practice/atcoder/ABC/014/src/c.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)
	arr := make([]int, 1000010)
	for i := 0; i < n; i++ {
		var a, b int
		fmt.Scan(&a)
		fmt.Scan(&b)
		arr[a] += 1
		arr[b+1] -= 1
	}
	for i, _ := range arr {
		if i == 0 {
			continue
		}
		arr[i] += arr[i-1]
	}
	ans := 0
	for _, a := range arr {
		if a > ans {
			ans = a
		}
	}
	fmt.Println(ans)
}
